src/fetch_real_estate_prices.py

```diff
@@ -0,0 +1,126 @@
+import json
```

Comment:
I was in doubt about how to organize the imports, but didn't search about. Thanks for the tip.

Fixed in 0b48e05. But noting now, requests is a native module, correct? I didn't pay attention to this.